src/quo/components/wallet/keypair/view.cljs

```diff
@@ -101,8 +101,7 @@
                                               :selected?       selected?
                                               :container-style container-style
                                               :theme           theme))
-      :on-press       #(when (= action :selector) (on-press))
-      :pointer-events :box-only}
```

Comment:
Here I'm removing this `:pointer-events` prop because when it's set to `box-only` it will not allow the pointer events to hit reach the "..." menu button inside the key-pair component.

Perhaps we need to pass a configuration prop for this if we need to ensure it's `box-only` sometimes, thoughts?

Reply:
not sure why we need it at all 🤔

Reply:
I'm not sure why it's needed in the first place 🤔 .

Reply:
I added the `box-only` because when we use it for keypair selection, other non important areas are pressable and make hard to select the proper keypair.

@seanstrom Probably we should handle the `:box-only` according to the variant displayed

Reply:
@ulisesmac Oh I see, so this is meant to make it simpler to press the key-pair, in that case I'll try using `(when (= action :selector) :box-only)` for that prop. Thank you for clarifying 🙏

Reply:
@ulisesmac - I suppose the `on-press` should be handled by the selector (it receives the `on-change` prop), not the parent. WDYT? This would help us prevent these pointer event tricks.

https://github.com/status-im/status-mobile/blob/8aa1cadc663642adb257df499904a59a062039b1/src/quo/components/wallet/keypair/view.cljs#L52-L56

Reply:
@smohamedjavid I'm not sure, but I think the intent is to make the key-pair component easier to select during the key-pair selection step. If we put the `on-press` handler on the selector, that would make the target smaller for selecting a key-pair when ideally it would be the entire key-pair UI.